content/en/docs/tasks/configure-pod-container/quality-service-pod.md

```diff
@@ -26,11 +26,62 @@ scheduling and evicting Pods.
 
 ## QoS classes
 
-When Kubernetes creates a Pod it assigns one of these QoS classes to the Pod:
+Quality of Services classes classify pods into one of three types: Guaranteed, Burstable, and BestEffort.
+The classification of pods is based on the [resource limits and requests](/docs/concepts/configuration/manage-resources-containers/).
+Each classification has effects on the scheduling policy and Eviction policy of a pod as well as on the resource usage of nodes.
+
+### Guaranteed
+
+To be sure a pod will be included in the Guaranteed QoS, every container in the pod must have
+resource(memory and CPU) limits and requests set, and they both have to be the same. The scheduler will make sure that
+a pod running in a Guaranteed QoS class is assigned to a node that has enough resources to fulfill its resource requests.
+A pod running in a Guaranteed QoS class can make use of exclusive CPUs  thanks to the [static](/docs/tasks/administer-cluster/cpu-management-policies/#static-policy) CPU management policy.
+
+When a node becomes [out of resource](/docs/tasks/administer-cluster/out-of-resource/), QoS classification helps kubelet
+to prioritize pods when the node needs to reclaim resources or when `oom_killer` needs to kill containers.
+If the `Guaranteed` pod uses memory resource lower than the requested resource, the pod has the lowest
+priority to be [evicted](/docs/tasks/administer-cluster/out-of-resource/#evicting-end-user-pods) by kubelet.
```

Comment:
```suggestion
has the least chance to be [evicted](/docs/tasks/administer-cluster/out-of-resource/#evicting-end-user-pods) by kubelet.
```
@sftim I reword this sentence to add priority and ranking of pods to be killed by kubelet.